sql/moz-fx-data-marketing-prod/ga_derived/firefox_whatsnew_summary_v1/query.sql

```diff
@@ -0,0 +1,47 @@
+WITH wnp_visits AS (
+  SELECT
+    date,
+    visit_identifier,
+    hit_number,
+    bounces,
+    first_interaction,
+    trim(page_path_level1, '/') AS locale,
+    page_level_2 AS version,
+    country
+  FROM
+    `moz-fx-data-marketing-prod.ga_derived.www_site_hits_v1`
+  WHERE
+    date = '2021-01-01'
+    AND page_name LIKE '%/firefox/%/whatsnew%'
```

Comment:
Using `page_name` does avoid having to worry about the query string part of `page_path`, but with `page_name` I don't think you want to match if there's anything before `/firefox`.
```suggestion
    AND page_name LIKE '/firefox/%/whatsnew%'
```

Or using the `page_level_#` columns available in `www_site_hits_v1` this could be made even more precise:
```suggestion
    AND page_level_1 = "firefox"
    AND page_level_3 = "whatsnew"
```

Reply:
Tested the logic out and it seems much more precise like you suggested